src/chi-vue/package.json

```diff
@@ -26,6 +26,7 @@
     "directory": "src/chi-vue"
   },
   "dependencies": {
+    "@centurylink/webpack-dts-plugin": "file:webpack-dts-plugin-1.0.1-npm.tgz",
```

Comment:
the dependency is published in base repository
"@centurylink/webpack-dts-plugin": "1.0.1"

Reply:
Hi @cristb, if I don't put this dependence and add this dependence @centurylink/po-microapp-base": "1.7.2" in package.json of the chi-vue. I have this issue
![image](https://user-images.githubusercontent.com/42206552/145364923-c5ff07ea-08df-4a9e-8a47-083d8ddc4814.png)

The token that I use is authorized by the Centurylink organization. With the po-microapp-sample work well. This repo also uses this dependence.

Do you know which is the issue?

Thanks no much!!

Reply:
ok, sorry I hadn't understood correctly your comment.